
#define CONTROL_LANDED 0 // State when helicopter has landed successfully
#define CONTROL_START 1 // Start / Takeoff
#define CONTROL_TRIM 2 // Trim Servos
#define CONTROL_HOVER 3 // Hover
#define CONTROL_GOTOPOS 4 // Go to Position
#define CONTROL_TRAJECTORY 5 // Follow Trajectory
#define CONTROL_LANDING 6 // Landing maneuver